ads/eas.md

```diff
@@ -0,0 +1,55 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Cxense Display ad server (formerly Emediate Ad)
+
+## Supported parameters in the amp-ad tag
+
```

Comment:
Didn't mean the title. 
I understand that you want to put a table of supported parameters here, but if you take a look of the file you will find that the format is messed up. 

The easiest way is to get rid of the table. One example for you reference.
https://github.com/ampproject/amphtml/blob/master/ads/google/adsense.md